koji_containerbuild/cli.py

```diff
@@ -79,8 +85,9 @@ def handle_container_build(options, session, args):
     parser.add_option("--quiet", action="store_true",
                       help=_("Do not print the task information"),
                       default=options.quiet)
-    parser.add_option("--noprogress", action="store_true",
-                      help=_("Do not display progress of the upload"))
+    if not flatpak:
+        parser.add_option("--noprogress", action="store_true",
+                          help=_("Do not display progress of the upload"))
```

Comment:
Actually this doesn't work for container-build either.

Reply:
I was leaving container-build untouched more because i didn't want to change it in an unrelated patch than because I thought that it didn't have the same problems. But certainly this would leave the code in a confusing state. I'll redo the branch with a commit to fix up container-build prior to the addition of flatpack support.